### PR TITLE
Refactor: Switch from FA to Phosphor icons

### LINKED ISSUE #6511: 脱FontAwesome
## Summary

- FontAwesomeは無料で使えるものが少ない
- (無料だと)デザインがあまり好みではない

### PR DESCRIPTION
**Pull request by ThatOneCalculator**
# What
Switches from FontAwesome icons to [Phosphor icons](https://phosphoricons.com/)

# Why
Resolve #6511

# Additional Info 
WIP, about 20-30% done.

- [ ] Finish all icons
- [ ] Switch from unpkg to locally hosting

### CODE REVIEW

File: packages/backend/src/server/web/views/base.pug [hunk 1]
```diff
@@ -34,7 +34,8 @@ html
 		link(rel='prefetch' href='https://xn--931a.moe/assets/info.jpg')
 		link(rel='prefetch' href='https://xn--931a.moe/assets/not-found.jpg')
 		link(rel='prefetch' href='https://xn--931a.moe/assets/error.jpg')
-		link(rel='stylesheet' href='/assets/fontawesome/css/all.css')
+		//- link(rel='stylesheet' href='/assets/fontawesome/css/all.css')
```
Comment (ThatOneCalculator) [inline packages/backend/src/server/web/views/base.pug:37]: @syuilo Do you think you could provide some insight as to *where* this file (`/assets/fontawesome/css/all.css`) is? I can't seem to find it in the codebase.

Comment (acid-chicken) [reply packages/backend/src/server/web/views/base.pug:37]: https://github.com/misskey-dev/misskey/blob/8df2f19b5b371d73809d12b2ffbcba3ec3cffb2d/packages/backend/src/server/web/index.ts#L113
↓

https://github.com/misskey-dev/misskey/blob/8df2f19b5b371d73809d12b2ffbcba3ec3cffb2d/packages/backend/src/server/web/index.ts#L35
↓

https://github.com/misskey-dev/misskey/blob/8df2f19b5b371d73809d12b2ffbcba3ec3cffb2d/gulpfile.js#L23

**Comment by syuilo** [comment]
Thanks!
I am also attempting tabler-icons and have not yet decided which one to adopt, so it might be better to have your implementation stop for a moment.

**Comment by ThatOneCalculator** [comment]
Tabler has a very similar style to Phosphor, but Tabler only has 1900 icons while Phosphor has 7000. Also, Phosphor has reimplementations of the FA custom styles while Tabler does not iirc.

**Pull request closed without merging**